engines/scumm/verbs.cpp

```diff
@@ -1010,6 +1011,9 @@ void ScummEngine_v7::drawVerb(int verb, int mode) {
 		_charset->setCurID(vs->charset_nr);
 
 		// Compute the text rect
+		if (_language == Common::HE_ISR /* && !(_game.id == GID_FT && (_charset->getCurID() == 6 || _charset->getCurID() == 7))*/) {
```

Comment:
Is the commented out part going to be used in the final version?

Reply:
After checking again, I think it is safe to remove the commented out code (the final version will work without it)